### ISSUE TITLE
gatsby-theme-document is not installed via `aio plugins install`

### ISSUE THREAD

**Issue by shazron** [adobe/aio-cli-plugin-doc#1]
This works on a dev system (via aio plugins link) because the installation of sub-dependencies (via the theme being npm installed) is deferred until later, when the user runs `aio doc init` (part of the flow after the theme is copied).

However, when installing using `aio plugins:install @adobe/aio-cli-plugin-doc` (this published plugin) it fails to install the theme since it goes deep into the dependencies to install everything -- and failing -- so none of the commands work because we can't create from the theme that is missing.

## Estimated fix:
Include the peer dependencies as dependencies of this plugin.
```
    "gatsby": "^2.23.3",
    "react": "^16.13.1",
    "react-dom": "^16.13.1"
```

(see below for an alternative)

**Comment by shazron** [comment]
I'm including the theme as a dependency -- so the download occurs only once, and not per `aio doc init`. I may have to change the implementation to defer the downloading of the theme on the first run of `aio doc init` and cache it for subsequent runs.
